tardis/__init__.py

```diff
@@ -33,6 +33,100 @@
 logger.addHandler(console_handler)
 logging.getLogger("py.warnings").addHandler(console_handler)
 
+
+class FilterLog(object):
```

Comment:
should this be moved to another module, `util` maybe?

Reply:
Sure, but I was thinking that we can restructure this after this gets merged. I will create a new sub module with this Logger as well as the Numba logger for consistency & for easier tracking of the logging present in TARDIS 😄